CONTRIBUTE

```diff
@@ -0,0 +1,109 @@
+HOW TO CONTRIBUTE TO TQDM
+===========================
+
+This file describes how to contribute changes to the project, and how to 
+upload to the pypi repository. 
+
+
+HOW TO COMMIT YOUR CONTRIBUTIONS 
+----------------------------------------------------------- 
+
+Contributions to the project is made using the `Fork & Pull` model. 
+Here's a quickstart on how to do that (your mileage may vary depending 
+on your git config): 
+
+- first create an account on github.
+- then go on the tqdm page `https://github.com/tqdm/tqdm` and fork it
+(click on the fork button).
+- now make a local clone of the project:
+`git clone https://github.com/your_account/tqdm.git`
+- do your changes on your local copy.
+- commit your changes `git commit -m "my message"`
+- TEST YOUR CHANGES (see below).
```

Comment:
Depends, I find myself often first trying to make new features, and then I try to fix anything that was broken. Trying to do both things at once can be pretty difficult, and you risk to face the "unit test immobility" (the fear to add new features because it can break stuff). If at least people test before pushing, they can always do a second commit to fix the regressions (and that's what I usually do).